src/Badge/Badge.js

```diff
@@ -13,10 +13,15 @@ export default function Badge({
   children,
   type = 'primary',
   style,
+  fontSize,
 }: BadgeProps) {
   return (
     <View style={[styles.wrapper, theme(type).wrapper, style]}>
-      <Text style={[styles.text, theme(type).text]}>{children}</Text>
+      <Text
+        style={[styles.text, theme(type).text, fontSize ? { fontSize } : {}]}
```

Comment:
I don't know if that would improve performance much, but I suppose that doing:
```
const dynamicStyle = StyleSheet.create({
   fontSize: {
       fontSize,
   },
});
```
would make sure that if existing rule for fontSize exists, it won't be recreated in memory, etc.

Reply:
this should be inside the render?

Reply:
ah you mean this
```
const theme = (fontSize: number) =>
  StyleSheet.create({
   fontSize: {
       fontSize,
   },
  });
```